### DISCUSSION TITLE
Sandbox Workbook Errors locally

### DISCUSSION THREAD

**Discussion by patrickjung712** [casact/chainladder-python#498] [General]
I'm new to chainladder and was working through the sandbox workbook locally. I get the following error on the below step:
```
>>> xyz_tri = cl.Triangle(
...     data=xyz_df,
...     origin="AccidentYear",       
...     development="DevelopmentYear",
...     columns=["Incurred", "Paid", "Reported", "Closed", "Premium"],
...     cumulative=True,
... )
C:\Python312\Lib\site-packages\chainladder\core\triangle.py:149: FutureWarning: 'A-DEC' is deprecated and will be removed in a future version, please use 'Y-DEC' instead.
```
Is this error and issue? or could it be causing the following issue?

I also get this following triangle summary
```
>>> xyz_tri
                                       Triangle Summary
Valuation:                                      2008-12
Grain:                                             OYDY
Shape:                                   (1, 5, 11, 19)
Index:                                          [Total]
Columns:    [Incurred, Paid, Reported, Closed, Premium]
```

The 19 columns includes several evaluation time periods that are a year - 1 month (such as 11, 23, 35, 59).

Am I doing something wrong? It works for me in the google colab workbook.

**Comment by kennethshsu** [comment]
Hi @patrickjung712, welcome!

I am unable to replicate the issue that you are experiencing. But since you are running the workbook locally, the most likely scenario is that you are working on a different version of `chainladder` or its dependencies.

What version of `chainladder` and `pandas` do you have on your local environment? You can run:
```python
import pandas as pd
import chainladder as cl
print(pd. __version__)
print(cl.__version__)
```

**Reply by patrickjung712** [reply]
Thanks. I have these versions:
``` python
>>> print(pd. __version__)
2.2.1
>>> print(cl. __version__)
0.8.18
```

**Reply by kennethshsu** [reply]
This is now resolved on the latest version of `chainladder` on GitHub, can you test it out?

There are still a bunch of warnings thrown, but at least the triangle object is correct now.

**Reply by patrickjung712** [reply]
Very cool. Yes it's all working for me now. Thanks!
